src/python/pants/core_tasks/register.py

```diff
@@ -50,6 +50,7 @@ def register_goals():
   Goal.register('lint', 'Find formatting errors in source code.')
   Goal.register('fmt', 'Autoformat source code.')
   Goal.register('buildozer', 'Manipulate BUILD files.')
+  Goal.register('meta-rename', 'Rename a target for its dependents')
```

Comment:
In Pants we use `dependencies` and `depeedees` to be more precise. 

`Rename a target and update its dependees' dependencies with the new target name`

cc @baroquebobcat if you have a better wording here.

Reply:
How about `'Rename a target'` in the core_task? That way it's concise along with the other ones. And then the string doc in `meta_rename.py` could expands and have `Rename a target and update its dependees' dependencies with the new target name ...`.